### ISSUE TITLE
❇️ `load_clean_sp()` functions additional formatting for arguments

### ISSUE THREAD

**Issue by mcuadera** [nish-kishore/sirfunctions#155]
ctry_name etc... arguments should be standardized inside the function

**Comment by mcuadera** [comment]
For this issue, it seems I was primarily referring to `load_clean_dist_sp()` which doesn't have a `dist_name` nor `prov_name` argument. Thoughts on the inclusion of these parameters? Additionally, we use either _ or . in the arguments. I can certainly deprecate the parameters using "." and replace with "_" as outlined [here](https://lifecycle.r-lib.org/articles/communicate.html#deprecate-an-argument-keeping-the-existing-default), but require feedback from the team.

**Comment by nish-kishore** [comment]
Yeah this sounds good, let's talk about use cases for dist_name/prov_name

**Comment by mcuadera** [comment]
Go ahead with the changes
